plugins/welcome-plugin/package.json

```diff
@@ -35,6 +35,9 @@
     "@theia/plugin-packager": "latest",
     "@eclipse-che/plugin": "0.0.1"
   },
+  "extensionDependencies": [
+    "Eclipse Che.@eclipse-che/workspace-plugin"
```

Comment:
should we fix our plug-ins ? so that extension dependencies looks better (without space) ?

Reply:
It's not a problem but to comply with the VS Code's rule for extensions IDs
> The id of an extension is always ${publisher}.${name}. For example: vscode.csharp.

we can consider renaming our plug-ins. I'll create an issue today for renaming and testing it.

Reply:
sorry, I meant matching the pattern
![Screenshot from 2020-10-27 23-03-39](https://user-images.githubusercontent.com/1636395/97361980-05285100-18a9-11eb-9016-a38b35590105.png)

Reply:
created the issue https://github.com/eclipse/che/issues/18220